brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Study (false) detections around bright star masks\n",
+    "The radius of current HSC bright star masks, made by Andy Goulding, are selected by comparing the density of detections in increasingly large annuli around each bright stars with the average detection density. \n",
+    "\n",
+    "Still, one of the identified problems with the HSC Public Data Release 2 is that false detections sometimes occur around the edge of these masks. The aim of this notebook is thus to provide a plot of the number of detections as a function of increasing radius, around bright stars and beyond the edge of their mask."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options\n",
+    "One might want to change the two cells below.\n",
+    "\n",
+    "The first one allows one to choose the path to the Butler repo we will read in, filter (note it only changes masks read in*, since we read detections from merged catalog), tracts, and binning options for the annuli.\n",
+    "\n",
+    "The rerun in `/project/mschmitz/hscRerun/mynobg` is a subset of the 25 central patches in tract 9697, and had local background subtraction turned off.\n",
```

Comment:
Please explain what the local background subtraction is and WHY you are turning it off for this analysis.  I thought you wanted to analyze the behavior of the (current) default pipeline?!

Reply:
I'll address this in the few sentences of conclusion. Long story short: I was (wrongly) expecting the plots to look much worse with the default pipeline. When they looked fine, we thought it might be because the local background subtraction removed (at least some of) the bright star halos, thus leading to no excess of detections. Turns out that's not the case, though.